extensions/quartz/deployment/src/test/java/io/quarkus/quartz/test/DisabledSchedulerTest.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.quartz.test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class DisabledSchedulerTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(Jobs.class)
+                    .addAsResource(new StringAsset("quarkus.quartz.enabled=false"),
+                            "application.properties"));
+
+    @Test
+    public void testNoSchedulerInvocations() throws InterruptedException {
+        // Check that @Scheduled method was never called
+        assertFalse(Jobs.LATCH.await(5, TimeUnit.SECONDS));
```

Comment:
This test will always take at least 5s :-(

Reply:
I reduced the wait time for the latch to 1.1 seconds. Not sure how to get under that time with `@Scheduled(every = "1s")`.

Reply:
Hm, maybe we should add `boolean isRunning()` to the `io.quarkus.scheduler.Scheduler` so that it would be easier to test the property. 

We should also clarify that `Scheduler#pause()` and `Scheduler#resume()` are no-op if the scheduler is disabled. And probably log a warning if it's disabled and someone attempts to call these methods.

Reply:
> Hm, maybe we should add `boolean isRunning()` to the `io.quarkus.scheduler.Scheduler` so that it would be easier to test the property.
Ok, should the isRunning value be set by the runtime config value in Scheduler instanciation (so test would basically only check the right config value)? Or should I also check that really no triggers have been created etc.?

Reply:
I'd say that it should always return `false` if `enabled=false`. Also don't forget that we have 2 impls of `Scheduler`. The default one and the quartz one. We need to think it over ;-).

Reply:
Not sure about QuartzScheduler, should we use something like scheduler.getCurrentlyExecutingJobs() > 0 to check if it (any job) is running? Or is a simple boolean check like in SimpleScheduler enough?

Reply:
That's a good question. Maybe we should use `org.quartz.Scheduler.isInStandbyMode()` but in that case we should modify the behavior of `QuartzScheduler.pause()` and `QuartzScheduler.resume()`. Currently, we call `pauseAll()/resumeAll()` but we should probably call `standby()/start()` instead. @machi1990 WDYT?

Reply:
And maybe another question :) How is it possible to inject the SimpleSchedule into my deployment test to check the isRunning flag? For QuartzScheduler it works because of @Produces.

Reply:
Let's do it.